.evergreen/failing-tests

```diff
@@ -0,0 +1,67 @@
+# This file lists known failing tests that for some reason or another cannot be
+# fixed and rather should be skipped in Evergreen builds.  Note that listing a
+# test in this file will result in the test being skipped on all platforms,
+# build variants, tasks, etc.
+#
+# The format of this file is as follows:
+# - one test per line, with an optional comment (total lines in file < 1,000)
```

Comment:
Setting an arbitrary limit seemed OK here, since it is a feature of the test runner.  It simplified the implementation of the parser since it meant not having to deal with deciding when we might have gotten into an infinite loop in parsing.